prow/integ-suite-ocp.sh

```diff
@@ -69,21 +77,45 @@ HUB="image-registry.openshift-image-registry.svc:5000/${NAMESPACE}"
 # Build the base command and store it in a variable.
 # TODO: execute the test by running make target. Do we need first to add a skip flag to the make target to be able to skip failing test on OCP
 # All the flags are needed to run the integration tests on OCP
+# Initialize base_cmd
+base_cmd=""
+
+# Set up test command and parameters
+setup_junit_report() {
+    export ISTIO_BIN="${GOPATH}/bin"
+    echo "ISTIO_BIN: ${ISTIO_BIN}"
+
+    JUNIT_REPORT=$(which go-junit-report 2>/dev/null)
```

Comment:
Using the same variable name JUNIT_REPORT for boolean above and here for a string is confusing

Reply:
Yes, that was not good. I changed the variable name to `TEST_OUTPUT_FORMAT` and default equal to `junit`